storybook/src/pages/amsterdam/ArticlePage/ArticleHeader.tsx

```diff
@@ -15,7 +15,7 @@ export const ArticleHeader = ({ heading, imageSrc }: ArticleHeaderProps) => (
         <Heading className="ams-mb-s" level={1}>
           {heading}
         </Heading>
-        <Paragraph size="small">7 juni 2024</Paragraph>
+        <Paragraph>7 juni 2024</Paragraph>
```

Comment:
This does not really relate to the ticket?

Reply:
Well, it’s the same kind of text at the other side of the heading, so it gets the same treatment.